### PR TITLE
[NO-JIRA] Remove old brand tokens

### PR DESCRIPTION
**Pull request by georgegillams**
Remember to include the following changes:
+ [x] `UNRELEASED.md`
+ [x] `README.md`
+ [x] Tests
+ [x] Docs (either update [backpack-docs](https://github.com/Skyscanner/backpack-docs) now, or create a follow up ticket)

### CODE REVIEW

File: packages/bpk-tokens/src/web/base/buttons.json [hunk 1]
```diff
@@ -397,23 +387,12 @@
     },
     "BUTTON_FEATURED_BACKGROUND_IMAGE": {
       "value": "{!SKY_BLUE}",
-      "type": "background-image",
-      "deprecated": true
+      "type": "background-image"
```
Comment (georgegillams) [inline packages/bpk-tokens/src/web/base/buttons.json]: I undeprecated this for consistency with the other button types

File: packages/bpk-tokens/src/web/base/calendar.json [hunk 2]
```diff
@@ -22,11 +22,6 @@
       "value": "{!SKY_BLUE}",
       "type": "color"
     },
-    "CALENDAR_DAY_BACKGROUND_COLOR": {
```
Comment (ojcurtis) [inline packages/bpk-tokens/src/web/base/calendar.json]: Lets double check this one first I feel this is being used

File: packages/bpk-tokens/src/web/base/typography.json [hunk 3]
```diff
@@ -13,18 +13,6 @@
       "type": "color",
       "category": "text-colors"
     },
-    "TEXT_COLOR_PRIMARY_LIGHT": {
```
Comment (ojcurtis) [inline packages/bpk-tokens/src/web/base/typography.json]: We may bring this back in the future - so might want to keep this right now?

File: packages/bpk-tokens/src/web/base/modals.json [hunk 4]
```diff
@@ -6,26 +6,6 @@
     "category": "modals"
   },
   "props": {
-    "MODAL_SCRIM_BACKGROUND_COLOR": {
```
Comment (ojcurtis) [inline packages/bpk-tokens/src/web/base/modals.json]: Can we check if this one is still in use?

**Pull request merged**